config-example.yml

```diff
@@ -5,6 +5,32 @@ graphhopper:
   # Local folder used by graphhopper to store its data
   graph.location: graph-cache
 
+  osm.import.accepted_highways: >
+    motorway,motorway_link,
+    trunk,trunk_link,
+    primary,primary_link,
+    secondary,secondary_link,
+    tertiary,tertiary_link,
+    unclassified,
+    residential,
+    living_street,
+    service,
+    pedestrian,
+    track,
+    bus_guideway,
+    escape,
+    raceway,
+    road,
+    busway,
+    footway,
+    bridleway,
+    steps,
+    corridor,
+    path,
+    cycleway,
+    proposed,
+    construction,
+    emergency_bay
```

Comment:
I already put these values as defaults in the corresponding Java code. So this is mostly here for illustration purposes and we should probably remove this again before we merge the PR. Explicitly putting such a list into the configuration file is only needed if one wants to reduce the size of the graph for performance reasons and maybe because of #1858 (see my other comment). 

One thing I don't like is that we cannot easily comment out values here, because YAML does not allow commenting out something from the middle of a literal (a multi-line string here). To comment out something we need to move the commented out values to the end of the list. We could use a list instead of this string, but so far our configuration only uses simple string key-values pairs (with the exception of profiles and profiles_ch/lm).

Reply:
I changed this and use an ignore list instead now.